src/main/java/com/ericsson/ei/subscriptionhandler/SubscriptionValidator.java

```diff
@@ -121,24 +121,34 @@ private static void validateNotificationMessageKeyValues(List<NotificationMessag
 
     /**
      * Validation of notificationMeta parameter Throws
-     * SubscriptionValidationException if validation of the parameter fails due to
-     * wrong format of parameter.
+     * SubscriptionValidationException if validation of the parameter fails due
```

Comment:
Maybe the Javadoc description should change a little now that the function has changed?